### PR TITLE
Restore smoke tests for notebooks

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/client/telemetry/index.ts [hunk 1]
```diff
@@ -811,7 +811,7 @@ export interface IEventNamePropertyMapping {
             | 'failed' // Installation disabled (this is what python extension returns).
             | 'install' // User chose install from prompt.
             | 'donotinstall' // User chose not to install from prompt.
-            | 'differentKernel' // User chose not to select a different kernel/interpreter.
+            | 'differentKernel' // User chose to select a different kernel.
```
Comment (DonJayamanne) [inline src/client/telemetry/index.ts:814]: @rchiodo /cc
Fixed https://github.com/microsoft/vscode-jupyter/pull/7514#discussion_r708504264

File: src/test/smoke/datascience.smoke.test.ts [hunk 2]
```diff
@@ -67,7 +67,6 @@ suite('Smoke Tests', () => {
     // }).timeout(timeoutForCellToRun);
 
     test('Run Cell in native editor', async () => {
-        return;
```
Comment (DonJayamanne) [inline src/test/smoke/datascience.smoke.test.ts:70]: Enable the test

File: .github/workflows/build-test.yml [hunk 3]
```diff
@@ -340,234 +340,6 @@ jobs:
           path: .nyc_output
           retention-days: 1
 
-  tests:
```
Comment (DonJayamanne) [inline .github/workflows/build-test.yml:343]: All of the tests have been commented out, hence absolutely no point in running a job that does nothing.

Comment (DonJayamanne) [reply .github/workflows/build-test.yml:343]: I.e. none of the functional tests work today (all have been commneted after webview was removed)

**Review by rchiodo — APPROVED** [review]

**Review by DavidKutu — APPROVED** [review]

**Review by joyceerhl — APPROVED** [review]

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**